<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>289</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>M64Py</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>&amp;Open ROM</string>
     </property>
     <addaction name="actionManually"/>
     <addaction name="actionFromList"/>
    </widget>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuStateSlot">
     <property name="title">
      <string>Change State Slot</string>
     </property>
    </widget>
    <addaction name="menuLoad"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionLoadState"/>
    <addaction name="actionSaveState"/>
    <addaction name="menuStateSlot"/>
    <addaction name="separator"/>
    <addaction name="actionLoadFrom"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionSaveScreenshot"/>
    <addaction name="separator"/>
    <addaction name="actionShowROMInfo"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPaths"/>
    <addaction name="actionEmulator"/>
    <addaction name="actionPlugins"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuEmulation">
    <property name="title">
     <string>&amp;Emulation</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionMute"/>
    <addaction name="separator"/>
    <addaction name="actionStop"/>
    <addaction name="actionReset"/>
    <addaction name="actionSoftReset"/>
    <addaction name="separator"/>
    <addaction name="actionLimitFPS"/>
    <addaction name="separator"/>
    <addaction name="actionSlowDown"/>
    <addaction name="actionSpeedUp"/>
    <addaction name="separator"/>
    <addaction name="actionCheats"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuWindowSize">
     <property name="title">
      <string>Window Size</string>
     </property>
     <addaction name="action1X"/>
     <addaction name="action2X"/>
     <addaction name="action3X"/>
    </widget>
    <addaction name="menuWindowSize"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulation"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar::item { border: 0px solid; }
QStatusBar { margin:0px; }</string>
   </property>
  </widget>
  <action name="actionManually">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_open_manually.png</normaloff>:/icons/action_open_manually.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manually...</string>
   </property>
  </action>
  <action name="actionFromList">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_open_from_list.png</normaloff>:/icons/action_open_from_list.png</iconset>
   </property>
   <property name="text">
    <string>From &amp;List...</string>
   </property>
  </action>
  <action name="actionKeyboard">
   <property name="text">
    <string>&amp;Keyboard</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_about.png</normaloff>:/icons/action_about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionClearHistory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Clear history</string>
   </property>
  </action>
  <action name="actionPaths">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_paths.png</normaloff>:/icons/action_paths.png</iconset>
   </property>
   <property name="text">
    <string>Paths</string>
   </property>
  </action>
  <action name="actionGraphic">
   <property name="text">
    <string>Graphics</string>
   </property>
  </action>
  <action name="actionEmulator">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_emulator.png</normaloff>:/icons/action_emulator.png</iconset>
   </property>
   <property name="text">
    <string>Emulator</string>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_plugins.png</normaloff>:/icons/action_plugins.png</iconset>
   </property>
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_license.png</normaloff>:/icons/action_license.png</iconset>
   </property>
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionLoadState">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_state_load.png</normaloff>:/icons/action_state_load.png</iconset>
   </property>
   <property name="text">
    <string>Load State</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionSaveState">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_state_save.png</normaloff>:/icons/action_state_save.png</iconset>
   </property>
   <property name="text">
    <string>Save State</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="slot0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="slot1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="slot2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="slot3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="slot4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="slot5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="slot6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="slot7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="slot8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="slot9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_quit.png</normaloff>:/icons/action_quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSaveScreenshot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_screenshot.png</normaloff>:/icons/action_screenshot.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Screenshot</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Resume</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_stop.png</normaloff>:/icons/action_stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionShowROMInfo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_rom_info.png</normaloff>:/icons/action_rom_info.png</iconset>
   </property>
   <property name="text">
    <string>Show ROM Info...</string>
   </property>
  </action>
  <action name="action1X">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </action>
  <action name="action2X">
   <property name="checkable">
    <bool>true</bool>
   </property>
  </action>
  <action name="action960x720">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>960x720</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_fullscreen.png</normaloff>:/icons/action_fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_reset.png</normaloff>:/icons/action_reset.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Mute</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionSlowDown">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_speed_down.png</normaloff>:/icons/action_speed_down.png</iconset>
   </property>
   <property name="text">
    <string>Slow Down</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionSpeedUp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_speed_up.png</normaloff>:/icons/action_speed_up.png</iconset>
   </property>
   <property name="text">
    <string>Speed Up</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action3X">
   <property name="checkable">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCheats">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_cheats.png</normaloff>:/icons/action_cheats.png</iconset>
   </property>
   <property name="text">
    <string>Cheats...</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionCustom"/>
  <action name="actionLoadFrom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_state_load.png</normaloff>:/icons/action_state_load.png</iconset>
   </property>
   <property name="text">
    <string>Load State...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_state_save.png</normaloff>:/icons/action_state_save.png</iconset>
   </property>
   <property name="text">
    <string>Save State As...</string>
   </property>
  </action>
  <action name="actionLimitFPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Limit FPS</string>
   </property>
  </action>
  <action name="actionSoftReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/action_reset.png</normaloff>:/icons/action_reset.png</iconset>
   </property>
   <property name="text">
    <string>Sof&amp;t Reset</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
